tools/wptserve/wptserve/request.py

```diff
@@ -365,11 +365,30 @@ def maybedecode(s):
     raise TypeError("Unexpected value in RequestHeaders: %r" % s)
 
 
+def maybeencode(s):
```

Comment:
Can you write a docstring to actually make it clear what this is and what this is for?